wolfcrypt/src/tfm.c

```diff
@@ -3076,12 +3076,9 @@ int mp_add_d(fp_int *a, fp_digit b, fp_int *c)
 #endif  /* HAVE_ECC || !NO_PWDBASED */
 
 
-#if defined(HAVE_ECC) || defined(WOLFSSL_KEY_GEN) || defined(HAVE_COMP_KEY)
```

Comment:
fp_s_rmap is only used if HAVE_ECC is defined.
Some compilers complain about unused static const data and others don't.

Reply:
Could you try running:

./configure --disable-ecc --enable-debug
./configure --disable-ecc --disable-fastmath --enable-debug

These are used when HAVE_ECC is not defined and debugging is enabled.